Business Social Network/AppModules/WishList/views/wishlist.html

```diff
@@ -26,10 +26,10 @@
     <div class="hovereffect">
         <img class="img-responsive default-img" src="/assets/WishList/noun_126735_cc.svg" alt="">
         <div class="overlay">
-           <h2 ng-bind="product.Name"></h2>
-           <h2>{{product.Price | currency: "&euro;"}}</h2>
+           <h2 ng-bind="product.name"></h2>
+           <h2>{{product.price | currency: "&euro;"}}</h2>
```

Comment:
@HazemElAgaty 
please use ng-bind instead of {{}} as it is not stable and will not usually yield the same view
http://stackoverflow.com/a/16126174